e2e/pom.xml

```diff
@@ -4,17 +4,13 @@
     <parent>
         <groupId>org.dhatim</groupId>
         <artifactId>fastexcel-parent</artifactId>
-        <version>0-SNAPSHOT</version>
+        <version>0.16.0-SNAPSHOT</version>
```

Comment:
Isn't this set during the publish process (and not updated via commit in the pom)?

Reply:
Yes and no. 
The version change (x.x.x-SNAPSHOT -> x.x.x -> x.x+1.x-SNAPSHOT) plus tagging, commits and publishing is totally handled b the Maven Release plugin.